experiments/openebs/openebs-pool-network-loss/openebs_pool_network_loss_k8s_job.yml

```diff
@@ -0,0 +1,79 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: openebs-pool-network-loss
+data:
+  parameters.yml: | 
+# Refer docs for this data.
+  ---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: openebs-pool-network-loss- 
+spec:
+  template:
+    metadata:
+      labels:
+        name: openebs-pool-network-loss
+    spec:
+      serviceAccountName: %CHAOS_SERVICE_ACCOUNT%
+      restartPolicy: Never
+      containers:
+      - name: ansibletest
+        image: litmuschaos/ansible-runner:test
+        imagePullPolicy: Always
+        env: 
+          - name: ANSIBLE_STDOUT_CALLBACK
+            value: 'default'
+
+          # provide openebs namespace
+          - name: OPENEBS_NAMESPACE
+            value: ''
```

Comment:
```suggestion
            value: 'openebs'
```

Reply:
done

Reply:
Make the `openebs` as default if user want to override they will pass the env